### PR TITLE
DisableServiceAtomicOperation Refactor/Tests

### PR DESCRIPTION
**Pull request by dkirillov**
@BrunoCarrier

### CODE REVIEW

File: clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/AbstractEcsAtomicOperation.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import com.netflix.spinnaker.clouddriver.aws.security.AmazonCredentials;
+import com.netflix.spinnaker.clouddriver.data.task.Task;
+import com.netflix.spinnaker.clouddriver.data.task.TaskRepository;
+import com.netflix.spinnaker.clouddriver.ecs.deploy.description.AbstractECSDescription;
+import com.netflix.spinnaker.clouddriver.ecs.services.ContainerInformationService;
+import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperation;
+import com.netflix.spinnaker.clouddriver.security.AccountCredentialsProvider;
+import org.springframework.beans.factory.annotation.Autowired;
+
+abstract class AbstractEcsAtomicOperation<T extends AbstractECSDescription, K> implements AtomicOperation<K> {
```
Comment (BrunoCarrier) [inline clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/AbstractEcsAtomicOperation.java]: make it public please

File: clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/validators/DisableServiceDescriptionValidator.java [hunk 2]
```diff
@@ -16,22 +16,32 @@
...
+public class DisableServiceDescriptionValidator extends CommonValidator {
 
   @Override
   public void validate(List priorDescriptions, Object description, Errors errors) {
+    DisableServiceDescription disableServiceDescription = (DisableServiceDescription) description;
 
-    // TODO - Implement this stub
+    boolean validCrednetials = validateCredentials(disableServiceDescription, "disableServiceDescription", errors, "credentials");
 
+    if (validCrednetials) {
```
Comment (BrunoCarrier) [inline clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/validators/DisableServiceDescriptionValidator.java]: Typo here

**Comment by BrunoCarrier** [comment]
+2 LGTM once you act on comments

File: clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/AbstractEcsAtomicOperation.java (hunk 1 above)
Comment (dkirillov) [reply clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/AbstractEcsAtomicOperation.java]: Done.

File: clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/validators/DisableServiceDescriptionValidator.java (hunk 2 above)
Comment (dkirillov) [reply clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/validators/DisableServiceDescriptionValidator.java]: Done.

**Pull request merged**